bootstrap/v2/pkg/utils/logging.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	log "github.com/sirupsen/logrus"
+)
+
+// PrettyPrint returns a pretty format output of any value.
+func PrettyPrint(value interface{}) string {
```

Comment:
Should we name this `PrettyPrintJSON` since it only works for json input?
In addition, you might find this library interesting: https://github.com/davecgh/go-spew

Reply:
It should work for pretty much any struct. Can you think of a struct it wouldn't work for?
In which case I'd vote for the shorter name.

Reply:
You're right, it seems like it will work for any type.
A side-effect is that json tags will affect the output (eg some fields may be omitted or named differently).
Not sure if we want this or not.
Since the name is PrettyPrint I assume it's ok.
Otherwise, if we want this for debugging, maybe we should use the `go-spew` library.

Reply:
I think its ok. Its only for pretty printing the output not debugging.